handbook/people-ops/buddy-program.md

```diff
@@ -0,0 +1,36 @@
+# Onboarding buddy program
+
+
+## Why we have onboarding buddies
+To ensure our new teammates are set up for success, we will pair them up with an “onboarding buddy” in order to become better acquainted with Sourcegraph team and culture. An onboarding buddy is an additional resource for new teammates who are learning how to navigate their new role and company.
+
+An onboarding buddy is essentially a new teammate’s first friend– someone to connect with that can share their experiences and advice to help make their transition a little smoother. A buddy will be the new teammate’s go-to person for questions (in addition to their manager and the People Ops team). They are someone to turn to with questions, someone who knows the ins and outs of the company and can offer help and guidance as the new teammate navigates their first few weeks at Sourcegraph. 
+
+## A great onboarding buddy will be
+**Approachable:** the new teammate is probably overwhelmed, let them know you can be accessed along with their manager if they have any questions or concerns along the way.
+
+**Proactive:** check-in and see how they’re doing, if they need help with anything. 
+
+**Compassionate:** remember what it was like to be new and tries to help ease that transition.
+
+**Helpful:** shows them the ropes of the organization and gives them tips on navigating it. 
+
+
+
+## Onboarding buddy logistics
+**Total length**
+
+- 3 months
+
+**Frequency**
```

Comment:
Curious if there's any new plan to check in or enforce this, or if it should rely on the buddy staying on top of it? Not sure we need anything here, it seems like a self-reinforcing system, but just wondering.

Reply:
I added a point to make myself responsible for setting up the recurring invites!